src/web-mercator-utils.js

```diff
@@ -164,41 +164,6 @@ export function addMetersToLngLat(lngLatZ, xyz) {
   return Number.isFinite(z0) || Number.isFinite(z) ? [newLngLat[0], newLngLat[1], newZ] : newLngLat;
 }
 
-/**
- * Calculates a mercator world position ("pixels" in given zoom level)
- * from a lng/lat and meterOffset
- */
-export function getWorldPosition({
-  longitude,
-  latitude,
-  zoom,
-  scale,
-  meterOffset,
-  distanceScales = null
-}) {
-  // Calculate scale from zoom if not provided
-  scale = scale !== undefined ? scale : zoomToScale(zoom);
-
-  // Make a centered version of the matrix for projection modes without an offset
-  const center2d = lngLatToWorld([longitude, latitude], scale);
-  const center = new Vector3(center2d[0], center2d[1], 0);
-
-  if (meterOffset) {
```

Comment:
Just a note: The `meterOffset` logic is used by/intended to be used by first person viewports/controllers to resolve meter offsets on top of lng/lats. Assume this will still supported in the deck.gl replacement.